File: Course_Exam/Question2/Course.py
class Course:
    def __init__(self, courseName, teacherName):
        self.courseName = courseName
        self.teacherName = teacherName

    def getInfo(self):
        a =1;
        for x in self:
            print('|%d|    %-30s|%30s|' % (a, x.courseName, x.teacherName))
            a += 1

File: Course_Exam/Question1/src/Course.java
public class Course {
    public String courseName;
    public String teacherName;

    public Course(String courseName, String teacherName) {
        this.courseName = courseName;
        this.teacherName = teacherName;
    }

    public String getCourseName() {
        return courseName;
    }

    public void setCourseName(String courseName) {
        this.courseName = courseName;
    }

    public String getTeacherName() {
        return teacherName;
    }

    public void setTeacherName(String teacherName) {
        teacherName = teacherName;
    }

    public static void  getInfo(Course[] y) {
        int a = 0;
        for(Course  x : y) {

            System.out.printf("|%d|%40s|%40s|%n", ++a, x.getCourseName(), x.getTeacherName());

        }
    }
}
